README.md

```diff
@@ -8,7 +8,7 @@ MinIO Client SDK provides higher level APIs for MinIO and Amazon S3 compatible c
 
 ## Install from NuGet
 
-To install MinIO .NET package, run the following command in Nuget Package Manager Console.
+To install [MinIO](https://www.nuget.org/packages/Minio/) .NET package, run the following command in Nuget Package Manager Console.
```

Comment:
Can this be `To install [MinIO .NET package](https://www.nuget.org/packages/Minio/)` ?

Reply:
done